CSharpExercises/Session03/Rohan/PayrollManagement/PayrollManagement/Employee.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PayrollManagement
+{
+    internal abstract class Employee
+    {
+        public string OrganizationName()
```

Comment:
Organization Name is a property. Not a method.

Reply:
public static string OrganizationName {get; set;}

//Consumption
Employee.OrganizationName  = "XYZ Company";